README.md

```diff
@@ -21,22 +21,25 @@ Local   | `aurora.test.near`  | 1313161556 | 2.6.1
 [Mainnet]: https://explorer.near.org/accounts/aurora
 [Testnet]: https://explorer.testnet.near.org/accounts/aurora
 
-## Prerequisites
+## Development
 
-### Prerequisites for Building
+### Prerequisites
 
+- Node.js (v14+)
 - Rust nightly (2021-03-25) with the WebAssembly toolchain
-- GNU Make (3.81+)
+- cargo-make
 
 ```sh
 rustup install nightly-2021-03-25
 rustup target add wasm32-unknown-unknown --toolchain nightly-2021-03-25
+cargo install --force cargo-make
```

Comment:
Why we still do not use stable Rust?

Reply:
rjson needs to be removed, it uses nightly features currently.